src/services/safe-config/safe-config.service.ts

```diff
@@ -2,22 +2,23 @@ import { HttpService } from '@nestjs/axios';
 import { Inject, Injectable } from '@nestjs/common';
 import { SafeConfigPage } from './entities/page.entity';
 import { SafeConfigChain } from './entities/chain.entity';
+import { HttpErrorHandler } from '../errors/http-error-handler';
 
 @Injectable()
 export class SafeConfigService {
   constructor(
     @Inject('SAFE_CONFIG_BASE_URL') private readonly baseUrl,
     private readonly httpService: HttpService,
+    private readonly httpErrorHandler: HttpErrorHandler,
   ) {}
 
   async getChains(): Promise<SafeConfigPage<SafeConfigChain>> {
     try {
       const url = this.baseUrl + '/api/v1/chains';
       const response = await this.httpService.axiosRef.get(url);
       return response.data;
-    } catch (error) {
-      console.log(error);
-      throw error;
+    } catch (err) {
+      this.httpErrorHandler.handle(err);
```

Comment:
This pattern will probably repeat in all GET endpoints, maybe it's worth making a helper function like
```js
getData<T>(path: string): Promise<T> {
  try {
     const url = this.baseUrl + path
     const response = await this.httpService.axiosRef.get(url);
     return response.data;
   } catch (err) {
     this.httpErrorHandler.handle(err);
   }
}
```

Reply:
I agree, I think it could be useful to have a separate layer in between `Service` and `HttpService` (something like `HttpClient`) which encapsulates this kind of functionality. So we can just call the `HttpClient` from any `Service` and reuse it. Maybe it's too much for this PR as it's just for logging, but I'd create an issue if @fmrsabino agrees 🙂

Reply:
Sure, it wasn't meant for this PR. Also you'll probably see larger patterns emerging eventually. I'll resolve this.

Reply:
I would not try to extract helpers at this stage. Let's see how things evolve first. It's still in the early stages.

@katspaugh the helper function that you proposed doesn't have any query parameters in place which we already use. Even thought it is an easy change that's one of the reasons why I prefer to not have helper functions right now.